src/test/scala/org/broadinstitute/dsde/workbench/sam/google/NewGoogleExtensionsSpec.scala

```diff
@@ -206,4 +210,168 @@ class NewGoogleExtensionsSpec(_system: ActorSystem)
       }
     }
   }
+  "GoogleExtensions: Pet Service Accounts" - {
```

Comment:
Added some pet tests to `NewGoogleExtensionsSpec` - this test is nice because it's easier to mock behavior thatn `GoogleExtensionsSpec`.